src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -182,6 +182,19 @@
       </IncludedFileForRunnerScript>
     </ItemGroup>
     
+    <!-- Introduce workaround for Dependency list to prevent copying local assemblies now that test csproj references pkgproj
+         This will remove all bin directory assemblies when TestWithLocalLibraries is false and local files are found.
+         This was added to allow the version of buildtools (that the buildtools repo depends on to be built) to be updated.  -->
+    <ItemGroup>
+      <IncludedFileForRunnerScriptRemove Include="@(IncludedFileForRunnerScript)"
```

Comment:
@weshaggard Included this line to prevent local dependencies from showing up in dependency list, which would break Helix runs. I started observing this after we started building the pkgproj reference in to the tests. This is a temporary workaround. Does this look ok at the moment?